timelock-server-distribution/build.gradle

```diff
@@ -16,3 +16,10 @@ distribution {
     args 'server', 'var/conf/timelock.yml'
     defaultJvmOpts "-Xms512m", "-Xmx512m", "-javaagent:service/lib/jetty-alpn-agent-${libVersions.jetty_alpn_agent}.jar"
 }
+
+task runTimeLock(type: JavaExec) {
+    classpath sourceSets.main.runtimeClasspath
+    main = "com.palantir.atlasdb.timelock.TimeLockServerLauncher"
+    jvmArgs = ["-javaagent:service/lib/jetty-alpn-agent-2.0.6.jar"]
+    args "server", "var/conf/timelock2.yml"
```

Comment:
should we name this something better? any reason it can't be `timelock.yml`?

Reply:
timelock.yml does not expose the HTTP/2 connector. This task is aimed at making it easier to test against the large internal product. Renaming it to `timelock-http2.yml`.